### ISSUE TITLE
Release coordination with MAXO?

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/maxo-annotations#45]
In HPO, we basically run the entire HPOA pipeline as part of every HPO release, and then releasing the HPOA files alongside the HPO release files. Was the plan not to do the same with MAXAN (Maxo Annotations), so we can use the MAXO versioning system with the annotations?

This would mean we need to somehow automate the process of obtaining the official mappings from POET, cc @iimpulse @pnrobinson

**Comment by iimpulse** [comment]
Yes there are 2 endpoints in POET. 1 to release the annotations (move approved annotations to official annotations), 2. Get the TSV output of official MAXAN.

**Comment by matentzn** [comment]
Great, thanks @iimpulse. Can you make a PR here:

https://github.com/monarch-initiative/MAxO/blob/master/src/ontology/maxo.Makefile

Similiar to what we do here: https://github.com/obophenotype/human-phenotype-ontology/blob/master/src/ontology/hp.Makefile#L445

That does the required processing?
